doc/developers/advanced_installation.rst

```diff
@@ -111,14 +113,14 @@ feature, code or documentation improvement).
 
 .. note::
 
-    You will have to run the ``pip install -v --no-use-pep517 --no-build-isolation -e .``
-    command every time the source code of a Cython file is updated
-    (ending in `.pyx` or `.pxd`). This can happen when you edit them or when you
-    use certain git commands such as `git pull`. Use the ``--no-build-isolation`` flag
-    to avoid compiling the whole project each time, only the files you have
-    modified. Include the ``--no-use-pep517`` flag because the ``--no-build-isolation``
-    option might not work otherwise (this is due to a bug which will be fixed in the
-    future).
+    Note `--config-settings editable-verbose=true` is recommended to avoid surprises
+    when you `import sklearn`. meson-python implements editable installs by recompiling
+    when executing `import sklearn`. Even changing Python files involves copying files
```

Comment:
I am not so sure anymore about "Even changing Python files involves copying files to the Meson build directory", I think it should be removed

Reply:
Removed it.